### PR TITLE
feat: Add November 2023 newsletter to the website

### LINKED ISSUE #1083: Add November 2023 newsletter to the website
## Issue Context

Every month, we try to get the newsletter up on the site within a week of emailing it. Currently, we're moving them over "by hand."

## Steps to update

In the code base, navigate to `app > routes > newsletter > issues` and create a new file `2023-11.jsx`.
You can look at the existing newsletters ( `app > routes > newsletter > issues`) as a template.

Make sure to add it to the index by following the steps in [Newsletters section in our README](https://github.com/Virtual-Coffee/virtualcoffee.io#newsletters) and update the content accordingly based on our email newsletter.

### PR DESCRIPTION
**Pull request by adiati98**
## Linked Issue

Closes #1083 

## Description

This PR adds November 2023 newsletter to the website and updates the `newsletters.ts`.

## Live Preview

https://deploy-preview-1084--virtual-coffee-io.netlify.app/newsletter/issues/2023-11

## Code of Conduct

> By submitting this pull request, you agree to follow our [Code of Conduct](https://virtualcoffee.io/code-of-conduct/)

### CODE REVIEW

**Comment by adiati98** [comment]
@danieltott in the email newsletter, we have a sponsor for this challenge. I've provided the paragraph in the `<LeadText>` on line 372-380. But I don't know where to store and how to add the image.

Below is the screenshot from our email newsletter:
 
blogging sponsor

File: app/routes/__frontend/newsletter/issues/2023-11.jsx [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+				<p>
+					Thanks to this issue's sponsor,{' '}
+					<a href="https://www.levelupfinancialplanning.com/?vc">
+						LevelUP Financial Planning
+					</a>
+					, who understands the importance of finding balance between having an
+					awesome life today, and being confident and excited about your future
+					possibilities.
+				</p>
+			</LeadText>
```
Comment (adiati98) [inline app/routes/__frontend/newsletter/issues/2023-11.jsx:382]: The description of the sponsor as written in the email.

**Review by adiati98 — COMMENTED** [review]
@danieltott in the email newsletter, we have a sponsor for this challenge. I've provided the paragraph in the `<LeadText>` here. But I don't know where to store and how to add the image.

Below is the screenshot from our email newsletter:
 
blogging sponsor

**Review by danieltott — APPROVED** [review]
@adiati98 image added 👍🏼 

Thank you!!

**Comment by adiati98** [comment]
Thank you, @danieltott!!!

**Pull request merged**